blog/2025-05-26-my-latest-toughts.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: my-latest-toughts
+title: Tapasztalataim a Docusaurus-ról
+authors: [zsofi]
+tags: [docusaurus, static-generator]
+---
+
+A statikus webhelygenerátorok, mint a Docusaurus, különösen erősek olyan esetekben, amikor a dokumentáció tartalma nem hozható létre automatikusan.
+
+<!-- truncate -->
+
+Az üzleti igény dokumentum a szoftverkészítés folyamán általában belsőleg jelenik meg – és ezért nem alkalmas arra hogy a Docusaurus-szal készítsük el. Ezenkívül a legtöbb szervezet már rendelkezik erre bejáratott módszerekkel.
+
+<!-- truncate -->
+
+A forráskód és/vagy az API dokumentációk általában automatikusan jönnek létre kódból, ezért nincs értelme a tartalmukat úgy átdolgozni, hogy az Docusaurus kompatibilis legyen.
```

Comment:
ez amúgy néhol vitatható - mi pl fenntartunk egy internal dokumentációs portált, ami docusaurussal fut. mi lényegében egy keretrendszert biztosítunk, ha a fejlesztők a generált API doksijaikat akarják valahol hostolni, így biztosítva egy egységes belső dokumentációs portált

lényegében ők csak egy script segítségével hozzáadnak egy git submodule-t, ahol található a specifikáció, ezt a script hozzáadja a konfigurációhoz. aztán automatikusan lefut egy build minden nap, így ha frissül az adott speci, úgy frissül a dokumentáció, ami hasznos tud lenni ha új fejlesztők jönnek, illetve a meglévőknek is, hogy van egy egységes hely ahol elérhető egyre több dokumentáció a belsős infrastruktúrának